addons/web_editor/static/src/js/editor/editor.js

```diff
@@ -178,56 +178,6 @@ var EditorMenuBar = Widget.extend({
         }
         return $.Deferred();
     },
-    /**
-     * @private
-     */
-    _saveCroppedImages: function () {
```

Comment:
Don't remove that method in stable, leave it and call `this.rte.saveCroppedImages` inside (and in fact, leave the code which calls it in the exact same way).